frontend/src/main/components/Users/UsersTable.js

```diff
@@ -60,6 +77,7 @@ export default function UsersTable({ users}) {
     const buttonColumn = [
         ...columns,
         ButtonColumn("toggle-admin", "primary", toggleAdminCallback, "UsersTable"),
+        ButtonColumn("toggle-driver", "primary", toggleDriverCallback, "UsersTable"),
     ]
 
     //const columnsToDisplay = showButtons ? buttonColumn : columns;
```

Comment:
Please remove commented out code when you see it.

Even if it's in the old code; if you touch a file, leave it cleaner than when you found it.